content/en/blog/_posts/grokkin-the-docs.md

```diff
@@ -0,0 +1,449 @@
+    part of the time problem. The other part is the high barrier to becoming a
+    reviewer or approver, much higher than what I've seen on other projects to
+    which I've contributed: Kubernetes member for at least three months, twenty
+    substantial PRs to become a reviewer, and thirty to become an approver.
+    Experienced open source Tech Writers who want to contribute to SIG Docs aren't
+    fast-tracked into approver and reviewer roles. On one hand, that high barrier
+    ensures that those roles are filled by folks with a minimum level of Kubernetes
+    documentation knowledge; on the other hand, it might deter experienced Tech
+    Writers from contributing at all... or from a company allocating a Tech Writer
+    to SIG Docs. Maybe SIG Docs should consider deviating from the Kubernetes
+    community requirements by lowering the barrier to becoming a reviewer and
+    approver -- on a case-by-case basis, of course.  
+- **Ensuring that Kubernetes terms, components, and objects have consistent naming across all pages**:
+    They should be identical to what is used in the Standardized Glossary; for
+    example, "apiserver"  vs "kube-apiserver" - being consistent reduces confusion.
+    Tracking these instances down is time-consuming but in the end is worth it for end users.
+- **Working with the Steering Committee, as well as kubernetes and kubernetes-sigs project owners, to create guidelines detailing which project docs belong in the Kubernetes docs vs the project's GitHub repo**:
+    The [Kubernetes Repository
+    Guidelines](https://github.com/kubernetes/community/blob/master/github-managemen
+    t/kubernetes-repositories.md) don't mention documentation at all. Some projects
+    have almost duplicate content, whereas others have conflicting content, between
+    the project's GitHub docs and the Kubernetes docs, and that just causes
+    confusion for end users. Clear guidelines would prevent that confusion. For
+    example, put roadmaps, milestones, and comprehensive feature details in the
+    `kubernetes/<project>` repo but installation, configuration, and usage details,
+    as well as tutorials, in the Kubernetes docs. These guidelines might be
+    different for projects in the kubernetes-sigs org, which, at first glance, have
+    more robust documentation.
+- **Removing content that is duplicated from, or can be found in, external sources**:
+    I've heard that the majority of Kubernetes users use Docker, so a good example
+    is Docker installation instructions. Rather than repeat what's in the Docker
+    docs, why not state which version of Docker works with which version of
+    Kubernetes, link to the Docker docs for installation, and mention any Kubernetes-specific
+    configuration? That idea is the same for any of the container runtimes that
+    Kubernetes supports. Why duplicate installation instructions for third-party
+    products?
+- **Removing third-party vendor content**:
```

Comment:
This one turned out to be disruptive enough to want a KEP. I _know_ there's a thing here about resisting the tide of vendor- or commercially-oriented pull requests that's just waiting to flow in if SIG Docs adopts a free-for-all policy.

I think if this were my article, that'd be the slant I took. What do you think @aimeeu ?

Reply:
Yes. Quite a bit has happened since the early drafts of this blog, so I do have to revisit this section.